### ISSUE TITLE
Accessibility review WCNL

### ISSUE THREAD

**Issue by rianrietveld** [WordPress/twentyseventeen#336]
My 2 cents on WordCamp Netherlands:
I love the theme, beautiful!

Note: I did not read all the issues in this repo, so maybe some issues are already addresses (sorry for that).

Links:
- outside entry-content there is no hover style on links
- the links in comments have no underline nor style on hover
- menu top level no style on hover

About the entry-meta in an archive, for example:

```
    <span class="posted-on">
        <span class="screen-reader-text">Posted on</span> 
        <a href="http://trunk.dev/2016/09/27/hello-world/" rel="bookmark">
            <time class="entry-date published" datetime="2016-09-27T17:06:23+00:00">September 27, 2016</time>
            <time class="updated" datetime="2016-10-12T11:38:03+00:00">October 12, 2016</time>
        </a>
    </span>
    <span class="byline"> by <span class="author vcard">
        <a class="url fn n" href="http://trunk.dev/author/webbeheer/">webbeheer</a>
    </span>
</span>
```

There is a link with as text the date of a post. This links goes to the post itself, and that link is repeated on the text for the post type. The date is a link to a post, and below that there is also a link to the same post.
On the single page this date is also a link to the age itself
IMHO this date text should not be a link at all, not on the archive and not on the single page.

After that is a link with the name of the author, that goes to the archive of the author.
Including the text "by" to the link is giving the link text more meaning/context .

Also if there are comments the time/date is the comments are also just a link to the date but in fact a link to the comment. Adding more context in the link text might be helpful to screen reader users.

Default header colour (minor issue, but it triggers a validation error)
The site title = #fff
the background = #fafafa
if images are disabled (or still loading) the site-title is not visible
the heading background can be adjusted by the user, but maybe it should be ok out of the box?

Pagination:
Add screen reader text or aria-label to the links to the pages to give more context to the link 
for example:  1 could be Page 1 (page hidden by screen reader text
and also indicate the active page
And add the text _Page_ to the next page arrow link

I think that the scroll down button should stay a button (disagree with https://github.com/laurelfulford/twentyseventeen/commit/bcd5af4d22aedafd97a80be1c97d8de14e132ca7)
the "link" doesn't go to a href, it's just an action to scroll the page down a but, so it should stay a button.

I hope this is helpful

**Comment by rianrietveld** [comment]
About the search form: 
Why choose this form? 

```
<form role="search" method="get" class="search-form" action="http://trunk.dev/">
    <label>
        <span class="screen-reader-text">Search for:</span>
        <input type="search" class="search-field" placeholder="Search …" value="" name="s" title="Search for:">
    </label>
    <button type="submit" class="search-submit"><span class="screen-reader-text">Search</span></button>
</form>
```

In several themes/frameworks the id issue has been solved by adding a unique id for the relation between input and label, generated with unique_id();

**Comment by davidakennedy** [comment]
Thanks for testing Twenty Seventeen @rianrietveld! I think most of these have been addressed:

> Links:
> 
> outside entry-content there is no hover style on links
> the links in comments have no underline nor style on hover
> menu top level no style on hover

These should be good, plus there is ongoing work to modify link styles a bit. See: https://github.com/WordPress/twentyseventeen/issues/240

> About the entry-meta in an archive

This was fixed by adding some context to the links as you suggest. We may be able to add more content to the author archive link. The date link needs to stay in case the page title is hidden in post formats, etc.

> Default header colour (minor issue, but it triggers a validation error)

It's only that color when a header image serves as the background. Not sure how that can be solved. In all other instances it's `#222` on `#fafafa`. The contrast should be sufficient enough on the default image that comes with the theme.

> Pagination

Will make these fixes! I think we can use Twenty Sixteen as a model.

> I think that the scroll down button should stay a button

I actually suggested this change. 😄  It now links to `#content` – do you think it should still be button?

> Why choose this form?

The theme has a custom search form because it uses an SVG in the button. The code was taken from Twenty Sixteen and is nearly identical. But I think that change should be easy enough.

**Comment by samikeijonen** [comment]
> It now links to #content – do you think it should still be button?

I'd say it should stay `<a href="#content"></a>` link for this reason.

**Comment by laurelfulford** [comment]
Merged #354 and #360, addressing the outstanding issues; closing. Thanks @rianrietveld!
